lib/update-credentials.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs');
+var path = require('path');
+var readline = require('readline');
+import { config } from './config';
+
+
+const createSphereCredentialsFile = (config) => {
```

Comment:
see above concerning `true,true,true`.   Maybe merge this script with the configuration loading into one and do all validation in one place? 

It's especially annoying because it overwrites the correct settings I had before.  

Plus "one more thing":  maybe we need to write another file in addition that suppports saving the API URL, too? if this here is the only persistence you have to seletively re-set apiUrl and authUrl in a new terminal, but not the other three.   Unfortunately the various CLIs documentation is confusing the hell out of me on how and where the save the API urls.

Reply:
Obsolete. file removed.

Reply:
currently we don't use any file and are using the node `config` to pass the arguments as parameters.